pennylane/interfaces/jax.py

```diff
@@ -95,7 +95,9 @@ def _execute(self, params, device):
         def wrapped_exec(params):
             exec_fn = partial(self.execute_device, device=device)
             return host_callback.call(
-                exec_fn, params, result_shape=jax.ShapeDtypeStruct((1,), JAXInterface.dtype)
+                exec_fn,
+                params,
+                result_shape=jax.ShapeDtypeStruct((1,), JAXInterface.dtype),

```

Comment:
Is JAX still a TODO, or it already skips the extra first pass?

Reply:
JAX and autograd use a keyword set in qnode initialization.  I had been setting it here but thought it better to add to the validation step.